package.json

```diff
@@ -60,11 +60,13 @@
   "devDependencies": {
```

Comment:
> package.json should also have a types field.

fixed.

> Most of our types are not in the main code, but in a types/ folder. Should we do that here too? Or does this make more sense?

I think `lib/` is a better choice for `micromark`. Because `micromark` has a relatively complex directory structure and it's easy to have problems if we maintain the same directory structure in both `lib/` and `types/`.

Reply:
Hmm. I still feel weird about having them in `lib/` as that is published only for debugging. Everybody is using the `dist/` code.

* I see your point about having the definitions close to the source, so writing/maintaining types in `lib/` makes sense
* Publishing the types in both `lib/` and `dist/` is superfluous
* In the published package, e.g., when debugging your node_modules, I think it makes more sense for them to live in `dist/`.

From these three points, could we have the source in `lib/`, copy them into `dist/` when building, and then in `package.json` ignore the ones in `lib/` but publish the ones in `dist/`?